app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -93,28 +97,36 @@ def destroy
 
   private
 
-    def find_service
-      @api_docs_service = current_account.api_docs_services.find_by_id_or_system_name!(params[:id])
-    end
+  def api_docs_params
+    params.require(:api_docs_service).permit(:name, :published, :description, :service_id, :body, :skip_swagger_validations)
+  end
 
-    def swagger_spec
-      if @api_docs_service.specification.swagger_2_0?
-        @api_docs_service.specification
-      else
-        resource_listing
-      end
-    end
+  def find_service_apis_collection
+    @service_apis = current_account.services
+  end
 
-    def resource_listing
-      {
-        swaggerVersion: "1.2",
-        apis: [
-          {
-            description: @api_docs_service.description.nil? ? @api_docs_service.name : @api_docs_service.description,
-            path: "#{admin_api_docs_service_path(@api_docs_service.system_name)}.{format}"
-          }
-        ],
-        basePath: "#{request.protocol}#{request_target_host}"
-      }
+  def find_service
+    @api_docs_service = current_account.api_docs_services.find_by_id_or_system_name!(params[:id])
+  end
+
+  def swagger_spec
+    if @api_docs_service.specification.swagger_2_0?
+      @api_docs_service.specification
+    else
+      resource_listing
     end
+  end
+
+  def resource_listing
```

Comment:
I didn't really touch this method except for removing an inconsistence of spaces 😄